app/src/components/market/sections/market_profile/market_status/closed.tsx

```diff
@@ -240,6 +254,45 @@ const Wrapper = (props: Props) => {
     setCurrentTab(newTab)
   }
 
+  const realitioAnswerNumber = Number(formatBigNumber(realitioAnswer || new BigNumber(0), 18))
+  const scalarLowNumber = Number(formatBigNumber(scalarLow || new BigNumber(0), 18))
+  const scalarHighNumber = Number(formatBigNumber(scalarHigh || new BigNumber(0), 18))
+
+  const finalAnswerPercentage =
+    realitioAnswer && realitioAnswer.eq('0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff')
```

Comment:
This value can be replaced by ethers . constants . MaxUint256
to make it more readable